src/vm/builtins.rs

```diff
@@ -642,8 +662,14 @@ impl<T: 'static> Forth<T> {
     pub fn add(&mut self) -> Result<(), Error> {
         let a = self.data_stack.try_pop()?;
         let b = self.data_stack.try_pop()?;
+
+        // NOTE: CURSED BECAUSE OF POINTER MATH
```

Comment:
you love to see it

Reply:
@hawkw fun story: https://cohost.org/jamesmunns/post/851945-oops-it-segfaults

Reply:
(also, this means ONLY "add" and "sub" are blessed to actually work with pointers, which might cause Fun Bonus Behavior later, open to suggestions here)

Reply:
oh no lol